run_job_mix.sh

```diff
@@ -138,19 +138,18 @@ run_orion_expr() {
     setup_orion_container ${device_id_arg} ${run_uuid_arg}
 
     # Run the experiment
-    # TODO: Fix duration
-    model_defn_string="${model_run_params[*]}"
-    ${DOCKER} exec -it ${ORION_CTR} bash -c \
+    duration=120
+    ${DOCKER} exec -it ${orion_ctr} bash -c \
         "LD_PRELOAD='/root/orion/src/cuda_capture/libinttemp.so' \
         python3.8 src/orion_scheduler.py \
         --device-type ${device_type} \
         --duration ${duration} \
-        ${model_defn_string}"
+        --model-details '${model_run_params_raw}'"
```

Comment:
All these changes are already in master. Something seems odd. 
https://github.com/aravindhank11/llama2-expr/blob/master/run_job_mix.sh